apps/web/src/app/(navigation)/games/edit-game/ListOfGamesSkeleton.tsx

```
@@ -0,0 +1,43 @@
+  CardHeader,
+  CardTitle,
+} from "@/components/ui/card";
+
+export default function ListOfGamesSkeleton() {
+  const skeletonCards = Array.from({ length: 5 });
+
+  return (
+    <>
+      {skeletonCards.map(() => {
+        return (
+          <Card className="mb-4 md:w-[383px] w-[250px]">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)